src/main/resources/db/migration/V212__mce_datafixes.sql

```diff
@@ -0,0 +1,179 @@
+
+UPDATE search_court
```

Comment:
Can you confirm that you have taken an extract from the prod database and analysed the differences between environments? I think you did, but just to confirm of course

Reply:
Cool thanks